LandSandBoat/server
# Re-align region IDs with what the client actually uses.

(as seen in ROM/165/78.dat)



**_I affirm:_**

## What does this pull request do?
https://gist.github.com/TeoTwawki/d46e51e761a95c6dffe11f0dc999f4d1

## Steps to test these changes
Tested as far as build and boot, Xaver was going to do some further tests for me. Potential issues anywhere the Lua does a greater than or less than check on regions, and where regions equal a bit in a mask.
